hwdb.d/parse_hwdb.py

```diff
@@ -181,6 +181,7 @@ def property_grammar():
              ('XKB_FIXED_MODEL', xkb_setting),
              ('KEYBOARD_LED_NUMLOCK', Literal('0')),
              ('KEYBOARD_LED_CAPSLOCK', Literal('0')),
+             ('WAKEUP_DEFAULT', Or((Literal('0'), Literal('1')))),
```

Comment:
You are checking `WAKEUP_DEFAULT` here while `WAKEUP_KEY_DEFAULT` is documented above?